ui/admin/config/environment.js

```diff
@@ -83,6 +83,10 @@ module.exports = function (environment) {
     'ember-cli-mirage': {
       directory: '../../addons/api/mirage',
     },
+
+    featureFlags: {
+      'primary-auth-method': false,
+    },
```

Comment:
Wondering if we should introduce feature flags in dev mode first and then move them upto production to avoid enabling them in prod by accident. Thoughts?